rust/src/address.rs

```diff
@@ -166,6 +167,35 @@ impl Deserialize for StakeCredential {
     }
 }
 
+#[wasm_bindgen]
+#[derive(Debug, Clone, Eq, Ord, PartialEq, PartialOrd, serde::Serialize, serde::Deserialize)]
+pub struct ByronAddressAttributes(pub (crate) legacy_address::Attributes);
+
+impl ByronAddressAttributes {
+}
+
+impl cbor_event::se::Serialize for ByronAddressAttributes {
```

Comment:
The (de)serialization logic for this is all incorrect because legacy_address doesn't follow the codgen format and instead it's a frankenstein of years of piling hacks instead of properly just generating byron support